osu.Game/OsuGameBase.cs

```diff
@@ -489,6 +498,9 @@ protected override void Dispose(bool isDisposing)
             LocalConfig?.Dispose();
 
             contextFactory.FlushConnections();
+
+            if (Host != null)
```

Comment:
I *believe* for extra robustness this check should be
```suggestion
            if (Host?.UpdateThread != null)
```

Reply:
Seems fine, but this scenario can never happen (due to order of execution in `GameHost`).

Reply:
I've applied this in #13618.